packages/round-manager/src/context/ApplicationContext.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { createContext, useContext, useEffect, useReducer } from "react";
+import { GrantApplication } from "../features/api/types";
+import { useWallet } from "../features/common/Auth";
+import { getApplicationById } from "../features/api/application";
+
+enum ActionType {
+  SET_APPLICATIONS = "SET_APPLICATIONS",
+  SET_LOADING = "SET_LOADING",
+  SET_ERROR_GET_APPLICATION = "SET_ERROR_GET_APPLICATION",
+}
+
+interface Action {
+  type: ActionType;
+  payload?: any;
+}
+
+type Dispatch = (action: Action) => void;
+
+export interface ApplicationState {
+  application?: GrantApplication;
```

Comment:
note: will followup after this PR to use ApplicationContext for lists of applications (eg ApplicationsReceived/Approved/Rejected tabs)